SECURITY.md

```diff
@@ -0,0 +1,13 @@
+# Reporting vulnerabilities
+
+Security vulnerabilities, or potential vulnerabilities, in NVIDIA
+packages should be reported to NVIDIA directly through
+[the NVIDIA Product Security](https://www.nvidia.com/en-us/security/) page.
```

Comment:
Maybe we should add the Nvidia forum `https://forums.developer.nvidia.com/` ?

Reply:
For security related stuff, probably better to just point people at the official page for those kinds of issues.